notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        # for our query, the coordinates must be a string composed of our translated RA and Dec.
</pre><p>Actually, you <em>can</em> pass an <code>astropy.coordinates</code>  object to the criteria! It's pretty convenient. Plus now you don't have to deal with the code that translates up above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>